client/src/partials/projects/projects.html

```diff
@@ -44,120 +40,188 @@
+                </div>
+              </div>
+
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.enterprise_id.$invalid }">
+                <label class="col-md-3 control-label">{{ "COLUMNS.ENTERPRISE" | translate }}</label>
+                <div class="col-md-9">
+                  <select class="form-control" name="enterprise_id" ng-model="ProjectCtrl.project.enterprise_id" id="bhima-enterprise-enterprise_id" ng-options="e.id as e.name for e in ProjectCtrl.enterprises | orderBy:'name'" required>
+                    <option value="" disabled> -- {{ 'SELECT.ENTERPRISE' | translate }} -- </option>
+                  </select>
+                </div>
+              </div>
+
+              <!-- zone de sante for snis -->
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.zs_id.$invalid }">
+                <label class="col-md-3 control-label">{{ 'COLUMNS.ZS' | translate }}</label>
+                <div class="col-md-9">
+                  <select class="form-control" name="zs_id" ng-model="ProjectCtrl.project.zs_id" id="bhima-project-zs" ng-options="zs.id as zs.format for zs in ProjectCtrl.zones | orderBy:'zone'" required>
+                    <option value="" disabled> -- {{ 'SELECT.ZS' | translate }} -- </option>
+                  </select>
+                </div>
+              </div>
+              <!-- end zone de sante for snis -->
+              <div class="form-group">
+                <label class="col-md-3 control-label">{{ 'COLUMNS.LOCKED' | translate }}</label>
+                <div class="col-md-9">                
+                  <input type="checkbox" id="locked" ng-true-value="1" ng-false-value="0" name="bhima-project-locked" ng-model="ProjectCtrl.project.locked">
+                </div>
+              </div>
+              <div class="form-group">
+                  <button class="btn btn-default" id="submit-project" type="submit" data-method="submit">
+                    {{ "FORM.SUBMIT" | translate }}
+                  </button>
+                  <button class="btn btn-default" type="button" ng-click="ProjectCtrl.cancel()" data-method="cancel">
+                    {{ "FORM.CANCEL" | translate }}
+                  </button>
+              </div>
+            </form>
+          </div>        
+        </div>
+
+        <div class="panel panel-primary" ng-switch-when="update">
+          <div class="panel-heading">{{ 'PROJECT.EDIT' | translate }}</div>
+          <div class="panel-body">
+            <form class="form-horizontal" name="UpdateForm" ng-submit="ProjectCtrl.submit(UpdateForm.$invalid)" novalidate>
+
+              <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.name.$invalid }">
+                <label class="col-xs-3 control-label required" for="bhima-project-name">{{ 'COLUMNS.NAME' | translate }}</label>
```

Comment:
The `required` class no longer does anything.  You should no longer use it.  Please remove it from any `<label>` element in this file that uses it.